stashed/src/components/user-controls/UserControls.scss

```diff
@@ -1,8 +1,8 @@
 .user-controls {
   color: #ffffff;
   position: absolute;
-  bottom: 0px;
-  left: 0px;
+  top: 0px;
```

Comment:
So it looks like you are trying to change files in the stashed/ folder 😄 This is actually the old version of the code that is no longer being used, we just have kept it around in case we need to pick out and reuse any helpful components or code. The only code you'll need to be working on is what's in the dwellinglyapp/src/ folder

Reply:
Hi Chris! So I took a look at the current code, and if I'm seeing correctly, it looks like moving the button is gonna take actual component refactoring, instead of just changing a CSS rule. The LogOutButton component is currently in Nav, and it looks like I'd have to move it to Header - which I'd be happy to do, but since I'm still new to this code base, I'm hesitant to make a change that large. 

Maybe this is a better issue for someone who's more familiar with the structure of the app? I just don't want to introduce bugs by moving a component to a place people aren't expecting. Or if you think it's safe to make the changes, I'm happy to do that too. 

Thanks!

Reply:
Alright I moved the logout button into the Header component, which should make styling easier. I tried to write new styles but something else must be overriding them.

It's where it needs to be now though, component-wise, and I made a note of where the new CSS rules need to go. I'll read up on Burma and SASS, but I wanted to get this request moving along, and see if anyone else knows which rules are affecting which components, and how to override them for this.

Alright!

Reply:
I don't see your changes, were you going to wait on committing them until the CSS was figured out? Feel free to reach out for assistance!